webapp/src/main/java/com/box/l10n/mojito/service/delta/PushPullRunCleanupService.java

```diff
@@ -0,0 +1,34 @@
+package com.box.l10n.mojito.service.delta;
+
+import com.box.l10n.mojito.service.assetExtraction.AssetExtractionCleanupJob;
+import com.box.l10n.mojito.service.pullrun.PullRunService;
+import com.box.l10n.mojito.service.pushrun.PushRunService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+
+/**
+ * @author garion
+ */
+@Service
+public class PushPullRunCleanupService {
```

Comment:
is this is not part of DeltaService, maybe rename the property `l10n.delta.cleanup-job.enabled` then?